documentation/guide/install-and-run.rst

```diff
@@ -0,0 +1,219 @@
+===========================
+Installing and running ZODB
+===========================
+
+This topic discusses some boring nitty-gritty details needed to
+actually run ZODB.
+
+Installation
+============
+
+Installation of ZODB is pretty straightforward using Python's
+packaging system. For example, using pip::
+
+  pip install ZODB
+
+You may need additional optional packages, such as `ZEO
+<https://pypi.python.org/pypi/ZEO>`_ or `RelStorage
+<https://pypi.python.org/pypi/RelStorage>`_, depending your deployment
+choices.
+
+Configuration
+=============
+
+You can set up ZODB in your application using either Python, or
+ZODB's configuration language.  For simple database setup, and
+especially for exploration, the Python APIs are sufficient.
+
+For more complex configurations, you'll probably find ZODB's
+configuration language easier to use.
```

Comment:
I agree, we started off using it but pretty quickly outgrew it. But it's still the default for the [Pyramid ZODB scaffold](http://docs.pylonsproject.org/projects/pyramid/en/latest/tutorials/wiki/installation.html#decisions-the-zodb-scaffold-has-made-for-you)/example, via way of [pyramid_zodbconn](http://docs.pylonsproject.org/projects/pyramid-zodbconn/en/latest/#using). So maybe it has some relevance for beginners.